integration_tests/minimal_project/readthedocs.yaml

```diff
@@ -0,0 +1,2 @@
+name: docs
```

Comment:
Looks like we aren't using this file, do we?

Reply:
If you look at https://github.com/rtfd/readthedocs-build/pull/48/files#diff-c5258b81d7a5c3182be62b7f82e02aedR105 there is a new test that uses it. Not sure if this is redundant considering the "find" functionality tests.

Reply:
The `apply_fs` function takes the dict and creates a filesystem with dirs and files on a temporal location.

Reply:
So, the files inside `integration_tests/` are used in other tests

Reply:
@stsewd maybe I'm confused: I'm thinking that the `apply_fs` function makes a temporary directory to test, and because I'd like to test whether the `.yaml` file is found I've added `readthedocs.yaml` to this folder and created the test to take this file and see if it's found. Am I missing something? Thanks

Reply:
> I've added readthedocs.yaml to this folder

If you mean by this line
https://github.com/rtfd/readthedocs-build/pull/48/files#diff-c5258b81d7a5c3182be62b7f82e02aedR52

Then, the file that you created isn't necessary, the apply_fs will create that file for you in a temporary directory with the content `name: docs\ntype: sphinx`. The file in `minimal_project` isn't necessary for this test.

Reply:
Thanks for explaining! I'll give it a quick try pre-work :)

Reply:
@stsewd tests passes :)